db/migrate/20140124124835_create_mezuro_configuration_ownerships.rb

```diff
@@ -1,8 +1,8 @@
-class CreateKalibroConfigurationOwnerships < ActiveRecord::Migration
+class CreateMezuroConfigurationOwnerships < ActiveRecord::Migration
   def change
-    create_table :kalibro_configuration_ownerships do |t|
+    create_table :mezuro_configuration_ownerships do |t|
       t.integer :user_id
-      t.integer :kalibro_configuration_id
+      t.integer :mezuro_configuration_id
 
```

Comment:
Yes, it's the old migration. Someone had replaced all instances of the text by mistake, and that means if you try to `rake db:migrate` from an empty database it will fail (because it will try to rename a column that doesn't exist).